src/app/interface-form/interface-form.component.ts

```diff
@@ -18,4 +18,8 @@ export class InterfaceFormComponent {
       Validators.max(this.userDeposit),
     ]),
   });
+
+  onSubmit(form: FormGroup) {
```

Comment:
You don't need to pass form since you have access to it right in a component class

Reply:
done